### ISSUE TITLE
Keyboard shortcut to send tweet is not disabled when alt text is missing

### ISSUE THREAD

**Issue by bnb** [aweary/twitter-required-alt-text#6]
currently if you command/control + enter while alt text is missing, a tweet will be sent.

OS: macOSS
Browser: Edge (Canary)
Views: Modal and inline tweet (maybe reply, too)

**Comment by aweary** [comment]
Nice, I didn't know about this shortcut! Maybe we can add an event listener for `keydown` and prevent it from propagating. I'll look into it.
